Documentation/ApiOverview/FlashMessages/Index.rst

```diff
@@ -76,7 +76,8 @@ or render it on your own where ever you want.
 This example adds the flash message at the top of modules when
 rendering the next request::
 
-   $messageQueue = $objectManager->get('\TYPO3\CMS\Core\Messaging\FlashMessageQueue');
+   $flashMessageService = $this->objectManager->get(FlashMessageService::class);
```

Comment:
This will only work if "FlashMessageService" is being aliased.

I would prefer writing `\TYPO3\CMS\Core\Messaging\FlashMessageService::class` instead.